ui/src/app/views/account-creation/component.tsx

```diff
@@ -32,6 +32,7 @@ import {
 import {Button} from 'app/components/buttons';
 import { FormSection } from 'app/components/forms';
 import {BoldHeader} from 'app/components/headers';
+import {profileApi} from 'app/services/swagger-fetch-clients';
```

Comment:
There should now be a bunch of unused imports from fetch and utils/fetch. If they aren't unused, then please replace other usages in this file as they shouldn't be needed.